server/app/services/applicant/ReadOnlyApplicantProgramServiceImpl.java

```diff
@@ -186,10 +186,14 @@ public ImmutableList<AnswerData> getSummaryData() {
         Optional<Long> updatedProgram = question.getUpdatedInProgramMetadata();
         Optional<String> originalFileName = Optional.empty();
         Optional<String> fileKey = Optional.empty();
+        Optional<String> encodedFileKey = Optional.empty();
         if (isAnswered && question.isFileUploadQuestion()) {
           FileUploadQuestion fileUploadQuestion = question.createFileUploadQuestion();
           originalFileName = fileUploadQuestion.getOriginalFileName();
           fileKey = fileUploadQuestion.getFileKeyValue();
```

Comment:
I think the fileKey variable can be inlined if we no longer need to call `setFileKey()` below.

E.g:
```
Optional<String> originalFileName = Optional.empty();
Optional<String> encodedFileKey = Optional.empty();
if (isAnswered && question.isFileUploadQuestion()) {
  FileUploadQuestion fileUploadQuestion = question.createFileUploadQuestion();
  originalFileName = fileUploadQuestion.getOriginalFileName();
  encodedFileKey = fileUploadQuestion.getFileKeyValue().map((fileKey) -> URLEncoder.encode(fileKey, StandardCharsets.UTF_8));
}
```